### PR TITLE
rfcdiff: modernize, test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by bfontaine** [comment]
Any reason not to merge this one?

File: Library/Formula/rfcdiff.rb [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+    url "https://tools.ietf.org/rfc/rfc43.txt"
+    sha1 "9f4c5dd7b20c4c62c31217b19d93a41054376bd8"
+  end
 
   def install
-    bin.install "rfcdiff"
+    # replace hard-coded paths
+    inreplace "Makefile.common", "/usr/share/man/man1", man1
+    # don"t use sudo
+    inreplace "Makefile.common", "sudo install", "install"
```
Comment (mikemcquaid) [inline Library/Formula/rfcdiff.rb]: Can we submit the above upstream (if there is one)?

File: Library/Formula/rfcdiff.rb [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
+                   "sources=rfcdiff.pyht",
+                   "prefix=#{prefix}",
+                   "install"
+  end
+
+  test do
+    rfcs = %w[rfc42 rfc43]
+    rfcs.each { |name| resource(name).stage name }
+
+    system "#{bin}/rfcdiff", *rfcs
```
Comment (mikemcquaid) [inline Library/Formula/rfcdiff.rb]: :+1:

File: Library/Formula/rfcdiff.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/rfcdiff.rb]: @mikemcquaid Yep, I sent a patch to upstream and included it in a `patch do` block.

**Pull request closed without merging**